backend/account/views.py

```diff
@@ -1,3 +1,28 @@
+from django.contrib.auth.models import User
 from django.shortcuts import render
+from .serializers import UserRegisterSerializer
+from rest_framework import generics, status
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
```

Comment:
remove unused import here and manage the import in this order everywhere;
- django import 
- rest_framwork import 
- custom import

Reply:
ok